src/api/cpp/cvc5_parser.cpp

```diff
@@ -86,6 +86,24 @@ const std::string& SymbolManager::getLogic() const
   CVC5_API_TRY_CATCH_END;
 }
 
+std::vector<Sort> SymbolManager::getDeclaredSorts() const
+{
+  CVC5_API_TRY_CATCH_BEGIN;
+  //////// all checks before this line
+  return d_sm->getModelDeclareSorts();
```

Comment:
Do we want to rename this internally, too?

Reply:
Sure.